cldf/forms.csv

```diff
@@ -2042,8 +2042,8 @@ Sakha-1-58-1,44953,Sakha,1-58,buːrɣa,buːrɣa,b uː r ɣ a,,19,true,^b uː r 
 Sakha-1-58-2,44954,Sakha,1-58,sillie,sillie,s i lː i e,,19,false,^s i ll i e$,Sakha,71312352364805278,,,5. no evidence for borrowing,0.0,"Pekarskij (1995 [1907-1930]: 2221) claims it is inherited from Turkic yel 'wind'; the ESTJ (1989: 174ff) doesn't include the Sakha form in their etymology, but they do say on p. 175 that it is sometimes compared to the Turkic form; Räsänen (1969: 195b) includes the Sakha word in the Turkic etymology.",,,unanalyzable,,1.0,,,,Proto-Turkic,1.0,,,,,,,,,,,
 Oroqen-1-58-1,36914,Oroqen,1-58,oŋko:,oŋko:,o ŋ k oː,,20,false,^o ŋ k o:$,Oroqen,79201353839164280,,The word oŋko: in Oroqen does not have the connotation of heavy rain accompanied by strong wind as the English word ‘storm’ would imply.,5. no evidence for borrowing,0.0,,,,unanalyzable,,1.0,,,,,0.5,,,,,,,,,,,
 Japanese-1-58-1,19755,Japanese,1-58,arashi,arashi,a r a ɕ i,,21,false,^a r a sh i$,Japanese,6122317035224499,嵐,,5. no evidence for borrowing,0.0,,,,semi-analyzable,ara-shi  rough-AR (disputed),0.75,,,,Old Japanese,1.0,,,,General,,0. No evidence for calquing,,,,wago,
-MandarinChinese-1-58-1,32161,MandarinChinese,1-58,bao4feng1yu3,bao4feng1yu3,p au ⁵¹ + f ə ŋ ⁵⁵ + ɥ y ²¹⁴,,22,false,^b ao 4 f eng 1 yu3 $,MandarinChinese,77161647939002505,暴风雨,,5. no evidence for borrowing,0.0,,,,analyzable compound,bao4-feng1-yu3 [burst.out - wind - rain],0.5,,2. Fairly common,,,0.5,,,,Formal,,0. No evidence for calquing,,,,,
-MandarinChinese-1-58-2,32162,MandarinChinese,1-58,kuang2feng1,kuang2feng1,kʰ u a ŋ ³⁵ + f ə ŋ ⁵⁵,,22,false,^k u a ng 2 f eng 1$,MandarinChinese,87272136640928082,狂风,"synonymous: da4feng1 大风 ,storm’, bao4feng1 暴风, feng1bao4 风暴 ,storm’",5. no evidence for borrowing,0.0,,,,analyzable compound,kuang2 - feng1 [wild/angry - wind],0.5,,,,,0.5,,,,,,,,,,,
+MandarinChinese-1-58-1,32161,MandarinChinese,1-58,bao4feng1yu3,bao4feng1yu3,p au ⁵¹ + f ə n k ⁵⁵ + ɥ y ²¹⁴,,22,false,^b ao 4 f en g 1 yu3 $,MandarinChinese,77161647939002505,暴风雨,,5. no evidence for borrowing,0.0,,,,analyzable compound,bao4-feng1-yu3 [burst.out - wind - rain],0.5,,2. Fairly common,,,0.5,,,,Formal,,0. No evidence for calquing,,,,,
```

Comment:
How do those changes arise? Are they confirmed to be correct:

Reply:
I assumed them to be a result of the most recent changes to the orthography profile of Mandarin Chinese. However, at that point I thought that the dataset hadn't been re-generated which I now notice it was. Will investigate.

Reply:
@LinguList Are there potentially any uncommited changes for the Mandarin Chinese profile to get the mappings as in the previous commit https://github.com/lexibank/wold/commit/fe81aac85cdf541a3f79a3a7e3d8acbe40f73b25? It looks like there were some additional changes to `eng` (as just `eng$` etc.).